docs/UserGuide.md

```diff
@@ -123,15 +123,17 @@ A help window will pop up containing basic introduction to the three core featur
 
 Adds a person to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [r/ROOM_NUMBER] [a/ADDRESS] [t/TAG]...`
+Format: `add n/NAME p/PHONE e/EMAIL [r/ROOM_NUMBER] [a/ADDRESS] [t/TAG]...`
```

Comment:
<img width="712" alt="image" src="https://github.com/user-attachments/assets/671b1778-860e-479a-abe7-9bdce997f8fb">

<img width="917" alt="image" src="https://github.com/user-attachments/assets/a8b30057-a282-4735-940f-5b4ff06ff238">

I think it mostly displays PHONE in the app, except for the help page?

Reply:
Oh ok, then we should just change the help page

Reply:
I thought this was decided when I asked for permission to standardise to PHONE in the feature specs?

Reply:
Yeah, but I noticed it was PHONE_NUMBER in the help and it was too late to change that before the release so I asked again in the group chat and changed to standardise. I think it's better to change back to PHONE though since the rest of the warnings are PHONE as well (which I didn't notice that day)